src/content/quickstart/gke/_index.md

```diff
@@ -0,0 +1,156 @@
+---
+date: 2020-11-13T17:55:18+01:00
+title: "Google Kubernetes Engine (GCP)"
+weight: 20
+---
+
+This quickstart guide covers the necessary steps to deploy two Google Kubernetes Engine (GKE) clusters on Google Cloud Platform (GCP) using `gcloud`. The clusters are named `cluster-a` and `cluster-b`. 
+
+Once the GKE clusters are deployed, we deploy Submariner with Service Discovery to interconnect the two clusters. Note that this guide focuses on Submariner deployment on clusters with non-overlapping Pod and Service CIDRs. For connecting clusters with overlapping CIDRs, please refer to the [Submariner with Globalnet guide](./globalnet/), which was not tested for this tutorial.
+
+{{% notice info %}}
+The guide assumes that you have a working GCP account and the `gcloud` binary installed and configured. To get the binary, please visit [Google's website](https://cloud.google.com/sdk/docs/install).
+{{% /notice %}}
+
+
+## Cluster creation
+Create two identical K8s clusters on GKE. For this demo, the following configuration was used:
+``` bash
```

Comment:
I think it's better if we can provide the whole example for just "copy&paste" for both cluster-a and cluster-b, 

Including the cidrs, etc: https://github.com/submariner-io/submariner-website/pull/33/files

so it's 'foolproof' :-)

Reply:
I see one problem with this: doesn't it depend on the subnets you have which IPs you can reserve?

Reply:
For me, Google was smart enough to generate non-conflicting CIDR ranges for both clusters

Reply:
Hmm, I don't know, I was able to do that in the past (the other GKE guide).

Otherwise we need to grab those later from gke manually, and use them in join, .. or firewall... manual process always hurts (at least for quickstarts/demos).

Reply:
ok, so we keep it in. I would say that some knowledge of GKE is required to follow the guide, and if the CIDRs are the problem users will figure it out very quickly :)